### ISSUE TITLE
False Positive - Protonmail

### ISSUE THREAD

**Issue by five2seven** [notracking/hosts-blocklists#898]
Please whitelist proton.me

Not sure how one of the best security-focused encrypted email providers ended up on the denylist

**Comment by zerkanz** [comment]
Absolutely no sense

**Comment by mcdazed** [comment]
Probably just an error, but agreed. Took me forever to find out why tf I couldn't load my inbox.

**Comment by naturalpb** [comment]
[Pull request created](https://github.com/notracking/hosts-blocklists/pull/899) to remove proton.me from the domains.txt blocklist.

**Comment by mpeter50** [comment]
It seems that from the list of blocklists this list gathers, proton.me is there on these 2 of them:
- https://filters.adtidy.org/extension/chromium/filters/7.txt
- https://easylist-downloads.adblockplus.org/easylist.txt

It would be interesting to know how did it end up on either of them, but I doubt that we will know. It is often very, very hard to follow where does a bad blocking entry come from.

**Comment by retupmoc99** [comment]
Its also on this list.  https://adaway.org/hosts.txt 

Just disabled this list from my filter and the site loads.  With it enabled its blocked. - I wasn't able to find it on the list but when the list is disabled the site resolves, so not sure maybe its referenced within there using another list?

**Comment by nylimited** [comment]
+1

Yep, all proton domains seem blocked. Can't even look at my account without unblocking.

**Comment by amano-kenji** [comment]
It seems agencies are trying to block proton by adding it on ad blocklists? Otherwise, proton couldn't have gotten on multiple ad blocklists so quickly. It couldn't have gotten on multiple blocklists in a day without a coordinated effort.

**Comment by Deanosim** [comment]
+1 just removed this list from Adguard Home because it suddenly started blocking Protonmail.

**Comment by mpeter50** [comment]
I don't know whether some kind of "agencies" are doing this, but I hate it that its so hard to track down where does a domain entry come from. Basically, it could be anyone.
Blocklist publishers like @notracking (but any other serious operation too) should publish the source code of the programs they use to make the block lists, and any data files they use with them (so the list sources too) so that the lists are reproducible locally. Preferably in the same repository.
This would allow for finding the source of a bad entry much easier by debugging the script, or with bigger changes to it.

Until its this hard to track down the source of bad listings (e.g. proton.me) who is doing this is almost impossible to know, they can continue freely with such changes (e.g. by continuing with piped.video, cryptpad.fr, and other such services), and they cant be accounted for it.
The ideal thing would be to contact the original source of the bad listings, and let them know of the problem. The only thing you can do, however, is remove the offending list that from your content blocker addon, like Deanosim did. Unless you have countless hours to days to try to track down the source.

This is quite an easy way to cause financial and reputational damage to any legit company, especially for those users who dont know why something like this might happen, and even more for users who are not yet familiar with the wrongly blocked service.

**Comment by notracking** [comment]
I'm very sorry for all of your inconvenience, I will be shutting down soon. See https://github.com/notracking/hosts-blocklists/issues/900 for more details.
